### PR TITLE
Adding .field & changing group error class name

### PR DESCRIPTION
**Pull request by BrentWMiller**
Added `.field` to our **dateTime** and **spinner** components.
- Will need some small fixes in the Theme.

I also changed `content-group` error messages to use the class name `.error-message` instead of just `.error`. This will make sure their styling is scoped better and not overwriting other element's classes.
- This change will require a Theme PR to change styling from `.error` to `.error-message`. Should just be an easy name switch.
#### This will all still work - just ugly without theme updates.

https://github.com/RenovoSolutions/RenovoTheme/pull/124

### CODE REVIEW

File: source/components/spinner/spinner.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <rl-generic-container selector="spinner.ngDisabled">
 	<template default>
-		<div ng-class="{ 'error': spinner.ngModel.$invalid }">
+		<div class="field" ng-class="{ 'error': spinner.ngModel.$invalid }">
```
Comment (KorinaGraber) [inline source/components/spinner/spinner.html:3]: Do you also need to set this on the disabled spinner below? I suppose validation styling in particular isn't very relevant if the spinner is disabled

Comment (BrentWMiller) [inline source/components/spinner/spinner.html:3]: Added it just to be safe.

Comment (KorinaGraber) [inline source/components/spinner/spinner.html:3]: Cool

**Comment by KorinaGraber** [comment]
:shipit:

**Pull request merged**